### ISSUE TITLE
Statistics between models

### ISSUE THREAD

**Issue by b3ade** [neulab/awesome#46]
Do you have any statistics or model performance between mBERT, model_without_co and model_with_co?
I see that you have alignment error rates on your models between aligners. So I'm wondering if you have any information on how your model is different from mBERT.

**Comment by zdou0830** [comment]
Hi, we essentially propose a way to extract alignments from mBERT and propose objectives to fine-tune mBERT on parallel data for word alignment. For the table in README, `Ours (w/o fine-tuning, softmax)` is the mBERT model performance using our alignment extraction method.
